src/runtime/entitlements-manager.js

```diff
@@ -203,7 +212,12 @@ export class EntitlementsManager {
    * @return {!Entitlements}
    */
   parseEntitlements(json) {
-    const isReadyToPay = json['isReadyToPay'];
+    const isReadyToPay = this.irtpStringToBoolean_(json['isReadyToPay']);
+    if (isReadyToPay == undefined) {
```

Comment:
This should be `x == null` - comparisons against `null` are more canonical for this.